src/commands/execute_command_utils.c

```diff
@@ -28,6 +28,8 @@ int	handle_builtins(int code, t_cmd_params *cmd)
 		return (ms_env(cmd));
 	else if (code == 7)
 		return (ms_exit(cmd));
+	else if (code == 9)
+		return (ms_alias(cmd));
```

Comment:
```suggestion
	else if (code == 8)
		return (ms_alias(cmd));
```
there will be a change if the translation pr is accepted before this pr